docs/reference/technology/mesh/wisun_intro.md

```diff
@@ -0,0 +1,36 @@
+#### Wi-SUN
+
+The Wi-SUN Field Area Network (FAN) is based on open standards from IEEE802, IETF, ANSI/TIA and ETSI.
+
+Wi-SUN FAN operates on license-exempt sub-GHz RF band and uses frequency hopping to lower interference. Therefore, Wi-SUN is well suited for outdoor installations and dense urban neighborhoods.
+
+The Mbed OS Wi-SUN stack is built on IPv6 over Low power Wireless Personal Area Networks (6LoWPAN), which itself builds on IEEE 802.15.4 to offer IP-based networking. Internet Protocol (IP) provides the core mechanism for relaying datagrams across IP networks, and its routing capabilities enable internetworking.
+
+You can find more details about Wi-SUN at the [Wi-SUN Alliance site](https://www.wi-sun.org).
+
+#### Mbed Wi-SUN API
+
+Mbed OS provides the [Mesh C++ API](../apis/mesh-api.html) for building Wi-SUN applications.
+
+- To connect to the Wi-SUN network, use the [Wi-SUN interface API](https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h).
```

Comment:
Query: Should we add a WisunInterface user API page? Or is this a background API that users don't really use?

Reply:
Isn't Wi-SUN API user API documented in https://github.com/ARMmbed/mbed-os-5-docs/pull/981?

Reply:
That has config and reference information. It's not a user API for WisunInterface. Our user APIs live here: https://github.com/ARMmbed/mbed-os-5-docs/tree/development/docs/api/networkinterfaces

They include a link to the Doxygen class and an example.